src/NLog/Layouts/JsonLayout.cs

```diff
@@ -108,12 +108,12 @@ protected override void InitializeLayout()
             base.InitializeLayout();
             if (IncludeMdc)
             {
-                ThreadAgnostic = false;
+                base.ThreadAgnostic = false;
             }
-#if !SILVERLIGHT && !NETSTANDARD || NETSTANDARD1_3
```

Comment:
@snakefoot the check ` !NETSTANDARD || NETSTANDARD1_3` is a bit strange, as every `NETSTANDARD` platfrom is also `NETSTANDARD1_3` in your project.json